test failing (test_get_word_lengths) for example.py

```python
example.py::test_reverse_words PASSED
example.py::test_get_word_lengths FAILED
=================================== FAILURES ===================================
____________________________ test_get_word_lengths _____________________________
    def test_get_word_lengths():
        text = "Three tomatoes are walking down the street"
>       assert get_word_lengths(text) == [5, 8, 3, 7, 4, 3, 6]
E       assert None == [5, 8, 3, 7, 4, 3, ...]
E        +  where None = get_word_lengths('Three tomatoes are walking down the street')
example.py:29: AssertionError
====================== 1 failed, 1 passed in 0.04 seconds ======================
The command "if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then py.test -vv example.py --cov example; else py.test -vv example.py; fi" exited with 1.
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,0 +1,3 @@
+# python stuff
+__pycache__
+.pytest_cache/

--- a/example.py
+++ b/example.py
@@ -19,8 +19,8 @@ def get_word_lengths(s):
     the word lengths in string s.
     """
     # uncomment next line in step 9
-#   return [len(word) for word in s.split()]
-    return None
+    return [len(word) for word in s.split()]
+    # return None
 
 
 # uncomment this function in step 6
@@ -34,4 +34,3 @@ def obscure_function():
     Example of a function that is never tested.
     """
     do_something_strange()
-

